apps/web/src/components/invites/MembersTable.tsx

```diff
@@ -6,6 +6,8 @@ import styled from 'styled-components';
 import * as capitalize from 'lodash.capitalize';
 import useStyles from '../../design-system/config/text.styles';
 import { MemberRole } from './MemberRole';
+import { When } from '../utils/When';
+import { useClipboard } from '@mantine/hooks';
```

Comment:
Could this also be placed above the relative imports? I have also opened a PR for the import-order eslint rule now 😅